<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DotPlotForm</class>
 <widget class="QWidget" name="DotPlotForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Word Size:</string>
       </property>
       <property name="buddy">
        <cstring>sb_wordSize</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_wordSize">
       <property name="suffix">
        <string> bits</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Window:</string>
       </property>
       <property name="buddy">
        <cstring>sb_windowSize</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_windowSize">
       <property name="suffix">
        <string> words</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>8000</number>
       </property>
       <property name="value">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Scale:</string>
       </property>
       <property name="buddy">
        <cstring>hs_scale</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hs_scale">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
